### DISCUSSION TITLE
3-D problem in Peridigm

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#244] [Q&A]
For a 3D problem analysis in peridigm, is it bond-based or state-based?

**Comment by sahirbutt** [comment]
The user has to specify what material model to use in the input (.xml or .yaml) script.
`Elastic` material is state-based and `Elastic Bond Based` material is bond-based. You can explore all available material models in the `src/materials` folder.

**Comment by ppandit95** [comment]
Hi @guangdongzhang 

The geometry in Peridigm is 3D in  all the examples as you can checkout.It is just that you have to restrain the geometry in certain dimension to get a 2D result.

Hope this helps.

Regards
Pushkar
